### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by mikeSimonson**
Refactor of the MigrateCommand to be able to test it.
And adding related tests.

### CODE REVIEW

File: lib/Doctrine/DBAL/Migrations/Tools/Console/Command/MigrateCommand.php [hunk 1]
```diff
@@ -153,4 +139,51 @@ public function execute(InputInterface $input, OutputInterface $output)
...
         }
     }
+
+    /**
+     * @param $question
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return bool
+     */
+    private function getConfirmationToExecute($question, InputInterface $input, OutputInterface $output)
```
Comment (chrisguitarguy) [inline lib/Doctrine/DBAL/Migrations/Tools/Console/Command/MigrateCommand.php]: I like  method names like `canExecute` for stuff like this. Just reads a bit cleaner in use.

```php
if (! $this->canExecute($question, $input, $output)) {
  // ...
}
```

Comment (mikeSimonson) [inline lib/Doctrine/DBAL/Migrations/Tools/Console/Command/MigrateCommand.php]: good idea

2015-06-04 16:08 GMT+02:00 Christopher Davis <notifications@github.com>:

**Comment by stof** [comment]
Please provide a description of what the PR does instead of just using ``Refactor``

**Comment by mikeSimonson** [comment]
@stof Do you know why those coveralls message keeps coming.

I deleted my repo from coveralls and revoked the coveralls token in the github admin.

**Pull request merged**